### PR TITLE
Normalize (try anyway) title strings

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

Inconsistencies in title strings

#### Changes

Try to normalize them

### CODE REVIEW

**Comment by t-nelson** [comment]
@CriesofCarrots mind taking a peek at these communication-wise?  If you have a few, the rest of the title strings could use a look as well (all in these four files).  The constraints I've been working on are:

1. ~16 char limit
1. Consistency between summaries
1. Clarity between fields in the same summary (eg. _new_ authority vs. _authorizing_ authority for `*-authorize`
1. Display of any field that the user can specify

File: libsol/system_instruction.c [hunk 1]
```diff
@@ -389,7 +389,7 @@ static int print_system_authorize_nonce_info(
     SummaryItem* item;
```
Comment (CriesofCarrots) [inline libsol/system_instruction.c:389]: On 351, `Advance Nonce` is the only system summary_primary_item in Title Case. (Commenting here, as github won't let me comment back that far)

File: libsol/stake_instruction.c [hunk 2]
```diff
@@ -367,14 +367,14 @@ static int print_stake_authorize_info(
...
+    summary_item_set_pubkey(item, "Set stake auth", info->account);
 
     switch (info->authorize) {
         case StakeAuthorizeStaker:
-            new_authority_title = "New stake auth.";
+            new_authority_title = "New stake auth";
             break;
         case StakeAuthorizeWithdrawer:
-            new_authority_title = "New w/d auth.";
+            new_authority_title = "New w/d auth";
```
Comment (CriesofCarrots) [inline libsol/stake_instruction.c:377]: On 503, similar screen reads "New withdraw auth". Not sure if 17 chars works there for some reason, but even if so, recommend that you change that one to be consistent.

File: libsol/vote_instruction.c [hunk 3]
```diff
@@ -298,14 +298,14 @@ int print_vote_initialize_info(
...
-        "New vote auth.",
+        "New vote auth",
         info->vote_init.vote_authority
     );
 
     item = transaction_summary_general_item();
     summary_item_set_pubkey(
         item,
-        "New withdraw auth.",
+        "New withdraw auth",
```
Comment (CriesofCarrots) [inline libsol/vote_instruction.c:308]: Same comment about withdraw consistency

File: libsol/system_instruction.c (hunk 1 above)
Comment (t-nelson) [reply libsol/system_instruction.c:389]: D'oh!  Missed one.  Nice catch!

File: libsol/stake_instruction.c (hunk 2 above)
Comment (t-nelson) [reply libsol/stake_instruction.c:377]: Looks like "withdraw" _just_ works here after removing the periods from "auth". It was butted up against the navigation arrows on both sides with.

FYI: 16 came from a full line of worst-case bold chars. "W" as per Ledger's page breaking logic.  We can get away with more in most cases

**Comment by t-nelson** [comment]
All addressed in a2afd5e421e498ef8fa363c32fba95a6f6f3f8c6

**Pull request merged**